src/app/server/Server.cpp

```diff
@@ -144,9 +148,7 @@ CHIP_ERROR Server::Init(const ServerInitParams & initParams)
         deviceInfoprovider->SetStorageDelegate(mDeviceStorage);
     }
 
-    err = mAccessControl.Init(initParams.accessDelegate, sDeviceTypeResolver);
-    SuccessOrExit(err);
-    Access::SetAccessControl(mAccessControl);
+    InitDataModelHandler(&mExchangeMgr);
```

Comment:
Did this get moved down because this way we don't queue events as we read the fabrics from storage?  If so, the ordering dependency should be documented here.  If not, why was it moved?